### DISCUSSION TITLE
Question: License

### DISCUSSION THREAD

**Discussion by TheDrawingCoder-Gamer** [FunkinCrew/Funkin#1553] [Q&A]
## What is your question?
A one liner license is a bad idea. I made a mod and wanted to have everyone who copied it to share alike, so i used Mozilla Public License. It requires all derivitives to be open source and use the same license, which is probably what you're looking for. A one liner is bound to get you in legal trouble.

**Comment by htv04** [comment]
It was originally Apache, but ninjamuffin changed it in #1106 as a joke and just hasn't changed it back/changed it to something else.

Since no code has changed between the license changes, it's probably fine to use the current code.

**Comment by TheDrawingCoder-Gamer** [comment]
:shrug: just saying even if it is a joke it's not a very good one when it comes to a license

**Comment by steviegt6** [comment]
> 🤷 just saying even if it is a joke it's not a very good one when it comes to a license

I believe ninja has been made well aware of that. Still waiting for him to do something, though.
